readme.md

```diff
@@ -731,6 +731,7 @@
 - [onetime](https://github.com/sindresorhus/onetime) - Only run a function once.
 - [mem](https://github.com/sindresorhus/mem) - Memoize functions - an optimization technique used to speed up consecutive function calls by caching the result of calls with identical input.
 - [require-uncached](https://github.com/sindresorhus/require-uncached) - Require a module bypassing the cache.
+- [node-eval] (https://github.com/gulp-bem/node-eval) - Eval JS-expression or require CommonJS modules and JSON files.
```

Comment:
The description is inaccurate. It makes it sound like it actually requires CommonJS modules.